scipy/fft/tests/test_helper.py

```diff
@@ -298,3 +336,153 @@ def test_errors(self):
                            match="invalid number of data points"
                            r" \(\[-2\]\) specified"):
             _init_nd_shape_and_axes(x, shape=-2, axes=None)
+
+
+class TestFFTShift:
```

Comment:
They are copied from `numpy` and modified for the array API.

The tests are new because the functions they are testing are new - previously, SciPy just called `np.fftshift` and so on.